### PR TITLE
Update help message in 'calicoctl get' for easier location of resource types

### LINKED ISSUE #1436: display valid resource types as list in `calicoctl get --help`
I always find it difficult to locate the valid resource types in `calicoctl get --help` output.

Can we move them into a bulleted list instead of a comma-separated list?

### PR DESCRIPTION
**Pull request by djosborne**
Fixes #1436

### CODE REVIEW

**Comment by caseydavenport** [comment]
@djosborne why about `delete`, etc?

I think there are other places we list these: https://github.com/djosborne/calico-containers/blob/4edfb739b5986ce5fe81e6b1ffd01fe5b02f8a6d/calicoctl/commands/delete.go#L69

**Comment by djosborne** [comment]
@caseydavenport PTAL. Added a bunch.

File: calicoctl/commands/apply.go [hunk 1]
```diff
@@ -48,8 +48,15 @@ Description:
   The apply command is used to create or replace a set of resources by filename
   or stdin.  JSON and YAML formats are accepted.
 
-  Valid resource types are node, bgpPeer, hostEndpoint, workloadEndpoint,
-  ipPool, policy, and profile.
+	Valid resource types are:
```
Comment (caseydavenport) [inline calicoctl/commands/apply.go]: Is this indentation right?  This is different from `get`

File: calicoctl/commands/get.go [hunk 2]
```diff
@@ -65,9 +65,17 @@ Description:
   or by type and identifiers.  JSON and YAML formats are accepted for file and
   stdin format.
 
-  Valid resource types are node, bgpPeer, hostEndpoint, workloadEndpoint,
-  ipPool, policy and profile.  The <TYPE> is case insensitive and may be
-  pluralized.
+  Valid resource types are:
+
+		* node
```
Comment (caseydavenport) [inline calicoctl/commands/get.go]: I think indentation of the list is wrong - shouldn't be tabs and shouldn't be indented so much!

**Comment by djosborne** [comment]
Gah sorry, I should have checked github before pinging you. looks like my editor's got a mind of its own

**Comment by caseydavenport** [comment]
LGTM, but I'd like to see the corresponding docs PR that goes with it.

**Comment by djosborne** [comment]
@caseydavenport gaze your eyes upon this: https://github.com/projectcalico/calico/pull/748

**Pull request merged**